.github/workflows/bundling-test.yml

```diff
@@ -34,6 +34,9 @@ jobs:
       - name: Build client library
         run: npm run build
 
+      - name: Ensure Node.js can load the built client library
+        run: node lib/index.js
```

Comment:
This is enough to check that module resolution is working in a plain Node+Js environment